src/HomeKit/HMHome.cs

```diff
@@ -80,5 +81,21 @@ public partial class HMHome
 
 			return GetServices (arr.ToArray ());
 		}
+
+		[NoTV]
+		[NoWatch]
+		[Introduced (PlatformName.iOS, 8,0, PlatformArchitecture.All, message: "This API in now prohibited on iOS. Use 'ManageUsers' instead.")]
```

Comment:
likely minor (ot sure if/how affects the display of the message): no need to duplicate the message in `[Introduced]`